### DISCUSSION TITLE
mspider/install/istio-update/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1475] [Announcements]
# mspider/install/istio-update/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/install/istio-update/

**Comment by Ye-Ting** [comment]
本页  【执行升级】 最后一张图不显示

**Comment by samzong** [comment]
升级的配图需要更新
